atd-vze/src/Components/GridTableFilterBadges.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from "react";
+import moment from "moment";
+
+import { Col, Row, Badge } from "reactstrap";
+
+const GridTableFilterBadges = ({
+  searchParams,
+  dateRangeParams,
+  advancedFilterParams,
+  advancedFiltersConfig,
+}) => {
+  const [searchBadgeText, setSearchBadgeText] = useState(null);
+  const [dateRangeBadgeText, setDateRangeBadgeText] = useState(null);
+  const [advancedFilterBadgeText, setAdvancedFilterBadgeText] = useState([]);
+
+  // Update search badge text
+  useEffect(() => {
+    const searchText = searchParams.value
+      ? `Search: ${searchParams.value}`
+      : null;
+
+    setSearchBadgeText(searchText);
+  }, [searchParams]);
+
+  // Update date range badge text
+  useEffect(() => {
+    const startDateRangeText = moment(dateRangeParams.startDate).format(
+      "MM/DD/YYYY"
+    );
+    const endDateRangeText = moment(dateRangeParams.endDate).format(
+      "MM/DD/YYYY"
+    );
+    const formattedDateRangeText = `Date: ${startDateRangeText} to ${endDateRangeText}`;
+
+    setDateRangeBadgeText(formattedDateRangeText);
+  }, [dateRangeParams]);
+
+  // Update advanced filter badges text
+  useEffect(() => {
+    // Collect string names of all filters applied
+    let advancedFilterText = [];
+    Object.entries(advancedFilterParams).forEach(([filter, boolean]) => {
+      boolean && advancedFilterText.push(filter);
+    });
+
+    const updateFiltersWithConfigLabels = filterTextArray => {
+      // Create dictionary of filter names and labels from
+      // filter configuration passed to GridTable
+      const filterDictionary = {};
+      Object.values(advancedFiltersConfig).forEach(filterGroup => {
+        filterGroup.filters.forEach(filter => {
+          filterDictionary[filter.id] = filter.label;
+        });
+      });
+
+      // Use dictionary to translate filters
+      return filterTextArray.map(filter => filterDictionary[filter]);
+    };
+
+    const humanReadableBadgeText = updateFiltersWithConfigLabels(
+      advancedFilterText
+    );
+    setAdvancedFilterBadgeText(humanReadableBadgeText);
+  }, [advancedFilterParams]);
+
+  return (
+    <Row className="mb-2">
+      <Col>
+        <span className="mr-1">Filters applied:</span>
```

Comment:
Looks great overall. Could we make it bigger though? Maybe Wring the `<span>` and the Badge components in an `<h3>` ?

Reply:
Yep, I agree with this since a date range will always be shown with the default of one year. I incorporated these changes along with @johnclary's feedback about the icons instead of text.

![Screen Shot 2019-10-31 at 2 04 33 PM](https://user-images.githubusercontent.com/37249039/67978154-79bda680-fbe7-11e9-8349-b7a35dbae4d3.png)